include/revng/Pipeline/ContainerFactorySet.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "llvm/ADT/StringMap.h"
+#include "llvm/ADT/StringRef.h"
+
+#include "revng/Pipeline/ContainerFactory.h"
+#include "revng/Pipeline/ContainerSet.h"
+
+namespace Pipeline {
+
+/// A container factory set is a collection of factories so that
+/// a set of empty containers can be spawned on demand.
+///
+/// We keep one of this for each Runner, so that the containers
+/// used by that runners can be created.
```

Comment:
wait, what is wrong about those? i thought the issue was the 80 columns size?

Reply:
You're using significantly less than 80 cols.
```suggestion
/// A container factory set is a collection of factories so that a set of empty                                                                                                               
/// containers can be spawned on demand.                                                                                                                                                      
///                                                                                                                                                                                           
/// We keep one of this for each Runner, so that the containers used by that                                                                                                                  
/// runners can be created.
```